fuzz/Cargo.toml

```diff
@@ -75,6 +76,12 @@ path = "fuzz_targets/iommu.rs"
 test = false
 doc = false
 
+[[bin]]
+name = "load_kernel"
```

Comment:
I think this (and the file) should be called `linux_loader` as that is what is fuzzing.

Reply:
Okay. I intended to make it differentiate from the other fuzzer from the `linux_loader`  crate (initially named as `loader_kernel_cmdline`). Now renamed both. PTAL.